### ISSUE TITLE
Can't get models working

### ISSUE THREAD

**Issue by kub3let** [go-skynet/LocalAI#111]
I'm trying to setup LocalAi, but no matter what model I use I always get errors.

downloaded from: https://gpt4all.io/models/ggml-gpt4all-j.bin
```
llama.cpp: loading model from /models/ggml-gpt4all-j
error loading model: unexpectedly reached end of file
llama_init_from_file: failed to load model
gptj_model_load: loading model from '/models/ggml-gpt4all-j' - please wait ...
gptj_model_load: n_vocab = 50400
gptj_model_load: n_ctx   = 2048
gptj_model_load: n_embd  = 4096
gptj_model_load: n_head  = 16
gptj_model_load: n_layer = 28
gptj_model_load: n_rot   = 64
gptj_model_load: f16     = 2
gptj_model_load: ggml ctx size = 5401.45 MB
gptj_model_load: memory_size =  1792.00 MB, n_mem = 57344
SIGILL: illegal instruction
PC=0x911789 m=3 sigcode=2
signal arrived during cgo execution
instruction bytes: 0x62 0xf2 0xfd 0x8 0x7c 0xc0 0x49 0x89 0x45 0x0 0x48 0x89 0x83 0x68 0x1 0x0
```

downloaded from: https://huggingface.co/eachadea/ggml-vicuna-13b-1.1/resolve/main/ggml-vic13b-uncensored-q8_0.bin
```
llama.cpp: loading model from /models/ggml-vic13b-uncensored-q8_0
error loading model: unrecognized tensor type 8

llama_init_from_file: failed to load model
gptj_model_load: invalid model file '/models/ggml-vic13b-uncensored-q8_0' (bad magic)
gptj_bootstrap: failed to load model from '/models/ggml-vic13b-uncensored-q8_0'
gpt2_model_load: invalid model file '/models/ggml-vic13b-uncensored-q8_0' (bad magic)
gpt2_bootstrap: failed to load model from '/models/ggml-vic13b-uncensored-q8_0'
stablelm_model_load: invalid model file '/models/ggml-vic13b-uncensored-q8_0' (bad magic)
stablelm_bootstrap: failed to load model from '/models/ggml-vic13b-uncensored-q8_0'
```

downloaded from: https://huggingface.co/eachadea/ggml-vicuna-13b-1.1/resolve/main/ggml-vic13b-uncensored-q5_1.bin
```
llama.cpp: loading model from /models/ggml-vic13b-uncensored-q5_1
error loading model: unrecognized tensor type 7

llama_init_from_file: failed to load model
gptj_model_load: invalid model file '/models/ggml-vic13b-uncensored-q5_1' (bad magic)
gptj_bootstrap: failed to load model from '/models/ggml-vic13b-uncensored-q5_1'
gpt2_model_load: invalid model file '/models/ggml-vic13b-uncensored-q5_1' (bad magic)
gpt2_bootstrap: failed to load model from '/models/ggml-vic13b-uncensored-q5_1'
stablelm_model_load: invalid model file '/models/ggml-vic13b-uncensored-q5_1' (bad magic)
stablelm_bootstrap: failed to load model from '/models/ggml-vic13b-uncensored-q5_1'
```

**Comment by JeshMate** [comment]
#88

**Comment by kub3let** [comment]
The CPU I'm running this on does support AVX, closing this in favor of #88

**Comment by kub3let** [comment]
The issue is fixed with latest 1.5.1 release
